docs/docsite/rst/scenario_guides/guide_meraki.rst

```diff
@@ -102,7 +102,7 @@ Returned Data Structures
 
 Meraki and its related Ansible modules return most information in the form of a list. For example, this is returned information by ``meraki_admin`` querying administrators. It returns a list even though there's only one.
 
-.. code-block:: yaml
+.. code-block::
```

Comment:
This should probably be: `.. code-block:: json`

Reply:
addressed in cf83f3d06f5067fe4b536299d871ea347b3ff8c0